core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/autoscales/AutoscaleV4Endpoint.java

```diff
@@ -108,6 +110,14 @@ public interface AutoscaleV4Endpoint {
     @ApiOperation(value = StackOpDescription.GET_BY_CRN, produces = APPLICATION_JSON, notes = Notes.STACK_NOTES, nickname = "getStackForAutoscale")
     StackV4Response get(@PathParam("crn") String crn);
 
+    @GET
+    @Path("/stack/crn/{crn}/dependentHostGroupsForAutoscaleHostGroups")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = StackOpDescription.GET_DEPENDENT_HOSTGROUPS_BY_CRN, produces = APPLICATION_JSON, notes = Notes.STACK_NOTES,
+            nickname = "getDependentHostGroupsForMultipleAutoscaleHostGroups")
+    DependentHostGroupsV4Response getDependentHostGroupsForMultipleHostGroups(@PathParam("crn") String crn,
+            @RequestBody @NotEmpty Set<String> hostGroups);
```

Comment:
Can the hostGroup list be part of parameters, instead of request body ?

Reply:
Added as @QueryParam